eng/scripts/Language-Settings.ps1

```diff
@@ -0,0 +1,56 @@
+$Language = "c"
+$Lang = "c"
+$PackageRepository = "C"
+$packagePattern = "*.json"
+$MetadataUri = ""
+
+function Extract-c-PkgProperties ($pkgPath, $serviceName, $pkgName)
+{
+  return null
+}
+
+# Parse out package publishing information given a nupkg ZIP format.
+function Parse-c-Package($pkg, $workingDirectory) 
```

Comment:
What is this function suppose to do here, since we don't have nupkg (NuGet packages). Should the comment be updated to mention vcpckg instead?